pkg/personname/groupInfo_test.go

```diff
@@ -0,0 +1,300 @@
+package personname
+
+import (
+	"errors"
+	"testing"
+)
+
+func checkGroupInfo(
+	t *testing.T,
+	expected GroupInfo,
+	expectedString string,
+	received GroupInfo,
+	group string,
```

Comment:
this is just a style nit: I usually don't see a lot of Go code with fn params newlined like this, even if they exceed 80chars. Go doesn't have great newline guidance AFAIK (it's something like "if it enhances readability you can do it." Most of the Go code I've seen doesn't do this, and I tend not to, but up to you as I don't feel very strongly.

Reply:
I'm pretty Draconian about staying within 88 characters in my own work (which is the default line length of python's Black formatter), but I also believe really strongly in matching the style of the codebase you are contributing to. If you prefer non-newlined parameters then I am more than happy to change it.

Inclination is to keep it, but happy to change it.

Reply:
Hmmm... I'm used to hard wrapping all my cpp code to 80chars due to various style guides I adhere to. However in go: https://golang.org/doc/effective_go.html#formatting says "Go has no line length limit. Don't worry about overflowing a punched card. If a line feels too long, wrap it and indent with an extra tab." 

This matches what I see in Go code at work and elsewhere so far in this library, and I rarely see fn signatures wrapped (as typically they should be short-ish anyway).

I think for consistency, in this case let's maybe put this on a single line for now, particularly as it's a test function and it doesn't feel altogether too long in a single line. If a function gets too many arguments, we may want to consider passing in structs instead. In the future, if we adopt a more strict line length convention, I can refactor the entire library to adhere to that style.

we can also consider batching arguments together by type which is 90 chars and very reasonable line length wise:

```go
func checkGroupInfo(t *testing.T, expected, received GroupInfo, expectedStr, group string)
```